repository/o.json

```diff
@@ -234,7 +241,18 @@
 			"releases": [
 				{
 					"sublime_text": "*",
-					"tags" : true
+					"tags": true
+				}
+			]
+		},
+		{
+			"name": "Okami – Adaptive Color Schemes",
+			"details": "https://github.com/Rayraegah/okami",
+			"labels": ["color scheme"],
+			"releases": [
+				{
+					"sublime_text": ">=3176",
```

Comment:
Unfortunately, this excludes everyone on the dev builds despite the latest being practically equivalent with the latest on the stable branch. I'll change it to 3153, where the new format was added.